lib/Elevate/Components.pm

```diff
@@ -75,11 +98,38 @@ our @BLOCKERS = qw{
   AutoSSL
 };
 
-push @BLOCKERS, 'Leapp';    # This blocker has to run last!
+push @CHECKS, 'Leapp';    # This blocker has to run last!
+
+# This is where to add new components that
+# have checks that are noops
+our @NOOP_CHECKS = qw{
```

Comment:
Why "no-op"? Seems like a strange choice of a term.

Reply:
Previously, there was a test that enforced that every blocker was included in the `BLOCKERS` array.  As part of this refactor, I changed the name of the array to `CHECKS`, but I did not want to remove the test enforcing that every component is included in this array, so I went with `NOOP_CHECKS` to try to convey that these checks should not actually do anything.  I'm definitely open to suggestion on a better way to do this, or we could just consider dropping the test, but my concern about dropping the test is that we end up creating blockers that never get executed.

As it is, I figured the lesser evil is just to enforce calling subs that simply return.